### ISSUE TITLE
mcpd error with existing persistence cookie profile

### ISSUE THREAD

**Issue by walterk82** [F5Networks/f5-appsvcs-extension#123]
##### Do you already have an issue opened with F5 support?
No

##### ISSUE TYPE

 - Bug Report

##### AS3 BUILD/ VERSION

```
3.13.0.3
```

##### BIGIP VERSION

```
14.1.0.6
```

##### SUMMARY

```
When specified a pre-existing session persistence profile that is configured in /Common (from legacy purposes). The AS3 declaration throws an error from mcpd that the profile can't be created.  However, the declaration indicates "bigip" and AS3 should not be creating this object just referencing it.
```

##### STEPS TO REPRODUCE

```
"persistenceMethods": [{
           "bigip": "/Common/Default-Cookie"
       }]
```

##### EXPECTED RESULTS

```
mcpd should not throw an error message in this case.
```

##### ACTUAL RESULTS

```
The following log message was produced in error.  The AS3 declaration processed fine.

err mcpd[526]: 01020066:3: The requested Persistence Profile (/Common/Default-Cookie) already exists in partition Common.
```

**Comment by walterk82** [comment]
Is there any timeline to address this?

**Comment by dstokesf5** [comment]
There is no timeline on this issue at this time.

**Comment by dstokesf5** [comment]
This issue was not resolved in 3.25.0. A different error, from trying to create a folder that already exists, was resolved in 3.25.0.

**Comment by walterk82** [comment]
This seems like a similar solution would apply not to just the internal AS3 objects but those referenced via "bigip" as well.

**Comment by dstokesf5** [comment]
This is resolved in the 3.34.0 release that is now available.
